view/frontend/web/js/pagseguro.js

```diff
@@ -333,11 +332,19 @@ RMPagSeguro.prototype.getInstallments = function(grandTotal, selectedInstallment
                     optionText += " (total R$" + (b[x].installmentAmount*b[x].quantity).toFixed(2).toString().replace('.', ',') + ")";
                 }
                 optionVal = b[x].quantity + "|" + b[x].installmentAmount;
-                if(b[x].quantity == selectedInstallment){
-                    parcelsDrop.append('<option value="'+optionVal+'" selected>'+optionText+'</option>');
-                }else{
+                // if(b[x].quantity == selectedInstallment){
+                //     parcelsDrop.append('<option value="'+optionVal+'" selected>'+optionText+'</option>');
+                // }else{
                     parcelsDrop.append('<option value="'+optionVal+'">'+optionText+'</option>');
-                }
+                // }
+            }
+            if(self.config.force_installments_selection != 1){
+                $('#rm_pagseguro_cc_cc_installments option[selected="selected"]').each(
```

Comment:
I guess that the use of `$(` instead of `jQuery(` is breaking the module.
After merging your branch into develop to test, **Credit Card method dissapeared from the checkout**.
I got the following error in the console:
[2020-10-26 12:05:56] [ERROR] Failed to load the "RicardoMartins_PagSeguro/js/view/payment/method-renderer/rm_pagseguro_ccmethod" component.

In further debug I got:
<img width="1636" alt="Finalizar_Compra" src="https://user-images.githubusercontent.com/191149/97124486-6276b200-1784-11eb-8330-5b67232d4662.png">